judgetuning/judge/judge_custom.py

```diff
@@ -0,0 +1,70 @@
+import json
+from typing import Callable
+
+from judgetuning.judge import Judge, JudgeAnnotation, AnnotationRequest
+
+
+class JudgeCustom(Judge):
+    def __init__(self, **kwargs):
+        self.kwargs = kwargs
```

Comment:
```suggestion
class JudgeCustom(Judge):
    """This class gives an example of a custom judge implementation to help you in case you want to benchmark your own judge."""
    def __init__(self, **kwargs):
        self.kwargs = kwargs
```